packages/client/src/pages/Game/core/entities/player/index.ts

```diff
@@ -61,6 +83,35 @@ export class Player extends GameObject {
     return this.lives === this.maxLives;
   }
 
+  public updateWeapon() {
+    this.weapon.setWeaponType = WeaponsList.Blaster;
+
+    if (this.idTimeout) {
+      clearTimeout(this.idTimeout);
+    }
+
+    this.idTimeout = setTimeout(() => {
+      this.weapon.setWeaponType = WeaponsList.Rocket;
+    }, 10000);
```

Comment:
```suggestion
    }, TIME_ACTIONS_ENHANCEMENT.blaster);
```
Сюда можно вставить TIME_ACTIONS_ENHANCEMENT из core/constants.ts